Language/Functions/Math/constrain.adoc

```diff
@@ -88,5 +88,14 @@ int constrainedInput = constrain(input, minimumValue, maximumValue);
 [float]
 === See also
 
+[role="definition"]
+* #DEFINITION# link:../../../variables/data-types/abs[abs]
+* #DEFINITION# link:../../../variables/data-types/map[map]
+* #DEFINITION# link:../../../variables/data-types/max[max]
+* #DEFINITION# link:../../../variables/data-types/min[min]
+* #DEFINITION# link:../../../variables/data-types/pow[pow]
+* #DEFINITION# link:../../../variables/data-types/sq[sq]
+* #DEFENITION# link:../../../variables/data-types/sqrt[sqrt]
```

Comment:
```suggestion
* #LANGUAGE# link:../abs[abs()]
* #LANGUAGE# link:../map[map()]
* #LANGUAGE# link:../max[max()]
* #LANGUAGE# link:../min[min()]
* #LANGUAGE# link:../pow[pow()]
* #LANGUAGE# link:../sq[sq()]
* #LANGUAGE# link:../sqrt[sqrt()]
```

- Correct links
- Use correct label: https://raw.githubusercontent.com/arduino/reference-en/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc
- Reproduce link text format from before the auto-link system was broken: https://web.archive.org/web/20190916201140/https://www.arduino.cc/reference/en/language/functions/math/abs/

Please make the equivalent fixes to all the other files in your PR.

Reply:
Hi @aliphys. I wanted to check back in with you. If you will update the pull request according to the instructions I provided above I will be happy to merge this important enhancement.

Reply:
Hi @aliphys. I wanted to check back in with you. If you will update the pull request according to the instructions I provided above I will be happy to merge this important enhancement.

Reply:
Hello @per1234 , Thanks for the (multiple) follow ups. Much appreciated. I will check in with @karlsoderby when he comes back to see what his plans for the reference pages are.

Reply:
Hi @aliphys. Any news?

Reply:
Just checked in with @karlsoderby , I'll make the changes you suggested by tomorrow.